### ISSUE TITLE
🐛 | Next track command is not navigating chapters in m4b audiobooks

### ISSUE THREAD

**Issue by tiffoknee** [MiczFlor/RPi-Jukebox-RFID#1926]
### What I did

I imported some m4b files

### What happened

the next track button doesn't navigate the chapters

### I expected this to happen

I expected the chapters to import as per @sandreas PR here: https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1057 and for the 'next track' command to skip to the next chapter.
The chapters are imported but next track just stops the track (as it's the only one in the folder)

### Further information that might help

VERSION] => 2.3 - 24258901 - master
    [CHAPTEREXTENSIONS] => mp4,m4a,m4b,m4r
    [CHAPTERMINDURATION] => 600
    [CMDVOLUP] => 118208207247158
    [CMDVOLDOWN] => 
    [CMDNEXT] => 1364201254187

########### SCRIPT playout_controls.sh (2022-12-18.15:45:39) ##
VAR COMMAND: getchapters
VAR VALUE: 
current file: /home/pi/RPi-Jukebox-RFID/shared/audiofolders/piggy_hero/Piggy Hero.m4b
chapters file: /home/pi/RPi-Jukebox-RFID/shared/audiofolders/piggy_hero/Piggy Hero.chapters.json
chapters for extension enabled: 1
chapters for duration enabled: 1
chapters file does exist, chapter count: 15

my syslog is a bit swamped with some unrelated issues..

``` Dec 18 14:31:59 raspberrypi systemd[1]: phoniebox-buttons-usb-encoder.service: Main process exited, code=exited, status=203/EXEC
Dec 18 14:31:59 raspberrypi systemd[1]: phoniebox-buttons-usb-encoder.service: Failed with result 'exit-code'.
Dec 18 14:32:09 raspberrypi systemd[1]: phoniebox-buttons-usb-encoder.service: Service RestartSec=10s expired, scheduling restart.
Dec 18 14:32:09 raspberrypi systemd[1]: phoniebox-buttons-usb-encoder.service: Scheduled restart job, restart counter is at 104.
Dec 18 14:32:09 raspberrypi systemd[1]: Stopped Phoniebox Buttons USB Encoder Service.
Dec 18 14:32:09 raspberrypi systemd[1]: Started Phoniebox Buttons USB Encoder Service.
Dec 18 14:32:09 raspberrypi systemd[9851]: phoniebox-buttons-usb-encoder.service: Failed to execute command: Permission denied
Dec 18 14:32:09 raspberrypi systemd[9851]: phoniebox-buttons-usb-encoder.service: Failed at step EXEC spawning /home/pi/RPi-Jukebox-RFID/components/controls/buttons_usb_encoder/buttons_usb_encoder.py: Permission denied
Dec 18 14:32:09 raspberrypi systemd[1]: phoniebox-buttons-usb-encoder.service: Main process exited, code=exited, status=203/EXEC
Dec 18 14:32:09 raspberrypi systemd[1]: phoniebox-buttons-usb-encoder.service: Failed with result 'exit-code'.
Dec 18 14:32:14 raspberrypi systemd[1]: Started Session 4 of user pi. 
```

### Base image and version
Raspbian GNU/Linux 10 (buster)

### Branch / Release
develop

### Installscript
`scripts/installscripts/buster-install-default.sh`

### RaspberryPi version
`3 B+`

**Comment by s-martin** [comment]
Are you using USB buttons?
If yes I assume the error messages in syslog are in fact related as the button commands don’t go through.

Do other button functions work?

**Comment by s-martin** [comment]
Closing as no interaction for a couple of months.
Feel free to reopen the issue, if the problem still occurs.

**Comment by TheNothingMan** [comment]
Sorry for posting to an issue that was closed years ago, but it seems to still persist.
I can confirm that on my build, using the GPIO buttons for next/previous track won't change chapters in m4b files. I noticed that there is an extra control in the web interface when playing files that contain chapters in one file.
Maybe I need to configure my buttons in a different way?
I would expect the buttons to do next/prev chapter IF chapters are available, and do next/prev track otherwise.

Any help would be appreciated - if further information or log files are needed, I would be glad to help if someone points me in the right direction.
